File: app/src/main/java/dev/ragnarok/fenrir/api/IOtherVkRetrofitProvider.java
package dev.ragnarok.fenrir.api;

import io.reactivex.rxjava3.core.Single;


public interface IOtherVkRetrofitProvider {
    Single<RetrofitWrapper> provideAuthRetrofit();

    Single<RetrofitWrapper> provideAuthServiceRetrofit();

    Single<RetrofitWrapper> provideLongpollRetrofit();

    Single<RetrofitWrapper> provideLocalServerRetrofit();
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/api/IOtherVkRetrofitProvider.kt
package dev.ragnarok.fenrir.api

import io.reactivex.rxjava3.core.Single

interface IOtherVkRetrofitProvider {
    fun provideAuthRetrofit(): Single<RetrofitWrapper>
    fun provideAuthServiceRetrofit(): Single<RetrofitWrapper>
    fun provideLongpollRetrofit(): Single<RetrofitWrapper>
    fun provideLocalServerRetrofit(): Single<RetrofitWrapper>
}